code/modules/mob/living/silicon/robot/robot_items.dm

```diff
@@ -404,3 +404,22 @@
 
 	user << "You fail to pick up \the [A] with \the [src]"
 	return
+
+/obj/item/weapon/tool/crowbar/robotic
```

Comment:
New robotic versions of tools. They have special sprites but are functionally identical